bench-run/publication/publish.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env python3
+
+import argparse
+import fnmatch
+import logging
+import os
+from typing import AnyStr
+from typing import List
+
+import requests
+
+from metrics import Metric
+
+URL = 'http://185.86.146.166:8086/api/v2/write?org=5d8890722b5f1318&bucket=metrics'  # NOQA
```

Comment:
I think we should configure it somehow to avoid hard-code. Maybe it is worth creating a special option for this stuff and placing the value in GitHub secrets.

Reply:
use secret